pepper-apis/src/main/java/org/broadinstitute/ddp/log/SlackAppender.java

```diff
@@ -109,17 +109,35 @@ private void init(String slackHookUrl,
      * Constructor used by deployments via logback.xml
      */
     public SlackAppender() {
-        init(System.getenv("SLACK_HOOK"),
-                System.getenv("SLACK_CHANNEL"),
-                System.getenv("SLACK_QUEUE_SIZE"),
-                System.getenv("SLACK_INTERVAL"));
```

Comment:
Ah ha! No wonder I couldn't find where we're configuring the `SlackAppender`. I carelessly thought that we intentionally removed it!

@pegahtah: I lied. We're still using `SlackAppender` and it's configured from config file.